### PR TITLE
release 0.0.175

### PR DESCRIPTION
**Pull request by ciaranRoche**
### Changes
- ba227a1 add new apis for addon config attribute
- 813ffe8 add list of requirements to addon parameter options
- 4710b21 Add Name fields to VPCs and Subnetworks
- 1defccd rename addon env object

Build Output - https://gist.github.com/ciaranRoche/8f6cb42071914f7dcd6049ee834d3b4a

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.175 Mar 01 2022
+- Add new apis for addon config attribute
+- Add list of requirements to addon parameter options
+- Add Name fields to VPCs and Subnetworks
+- Rename addon config env object
+
+## 0.0.174 Feb 16 2022
+- Add rhit_account_id to Account Class
```
Comment (zgalor) [inline CHANGES.md:12]: This PR documents 2 last releases?

Comment (ciaranRoche) [reply CHANGES.md:12]: Yeah 0.0.174 was not added to the changes doc, will I split it into 2 commits :thinking:

Comment (zgalor) [reply CHANGES.md:12]: Nevermind. 174 was already released

**Pull request merged**